.github/workflows/breadcrumbs.yml

```
@@ -0,0 +1,32 @@
+    branches: [ main ]
+    paths:
+      - 'pages/**/*.mdx'
+      - 'pages/**/*.md'
+
+jobs:
+  check-breadcrumbs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)